core/js/src/main/scala/com/softwaremill/sttp/simpleApi.scala

```diff
@@ -0,0 +1,5 @@
+package com.softwaremill.sttp
+
+object simpleApi extends SttpApi{
```

Comment:
I think I would be leaning towards calling this package `quick`. What do you think?

Reply:
sure